content/ar/docs/setup/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: دليل الاستخدام
+main_menu: true
+weight: 20
+content_type: concept
+no_list: true
+card:
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#بيئة-التعلم"
+    title: بيئة التعلم
+  - anchor: "#بيئة-الإنتاج"
+    title: بيئة الإنتاج  
+---
+
+<!-- نظره عامه -->
+
+ يعرض هذا القسم الطرق المختلفه لاعداد وتشفيل كوبرنيتس. إختر الطريقة المناسبة للتثبيت بناءً على متطلبات الصيانة والأمان والتحكم، والموارد المتاحة للتشغيل، والخبرة التقنية المتوفرة لإدارة العنقود.
+
+يمكنك [تنزيل كوبرنيتس](/releases/download/) لنشر عنقود كوبرنيتس على جهاز محلي، أو في السحابة، أو لمركز البيانات الخاص بك.
+
+يمكن أيضًا تشغيل عدة مكونات [كوبرنيتس](/docs/concepts/overview/components/) مثل {{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}} أو {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}} [كصور الحاويات](/releases/download/#container-images) داخل العنقود.
+
+**من المستحسن** تشغيل مكونات كوبرنيتس كصور حاويات أينما كان ذلك ممكنًا، والسماح لكوبرنيتس بإدارة تلك المكونات. لا يمكن تضمين المكونات التي تعمل على تشغيل الحاويات - على وجه الخصوص، kubelet - في هذه الفئة.
+
+
+إذا كنت لا تريد إدارة العنقود بنفسك، يمكنك اختيار خدمة مُدارة، بما في ذلك [المنصات المعتمدة](/docs/setup/production-environment/turnkey-solutions/). هناك أيضًا حلول موحدة ومخصصة أخرى عبر مجموعة واسعة من البيئات السحابية والمعدنية الخام.
+
+<!-- الجسم -->
```

Comment:
do not think we translate this

Reply:
what should be the solution here ? Do we ignore or refer to the english docs until we translate ?

Reply:
my bad coffee is still kicking in, I mistakened your comment line 😆